Appraisals

```diff
@@ -95,7 +95,7 @@ RAILS_VERSIONS = %w(
   4.2.5.1
   4.2.5.2
   4.2.6
-  5.0.0.beta3
+  5.0.0.1
```

Comment:
If things were done right it should have been tested **before** 4.2.7.1 and 5.0.0.1 was released.